### PR TITLE
[TASK] Support auto-registration of set labels from labels.xlf

### PR DESCRIPTION
**Pull request by bnf**
The set language files introduced in https://forge.typo3.org/issues/104831 are now auto-loaded from labels.xlf within the sets folder, if the file is provided and the set configuration does not reference an explicit labels xlf file.

### CODE REVIEW

**Comment by bnf** [comment]
The xlf handling introduced yesterday – https://github.com/TYPO3-Documentation/render-guides/pull/734 – is to be slightly extended to support autoloading.
Could you please review  & release, so that the following patch can be merged (I don't want to break documentation rendering, so I'm waiting until this is merged)
> 85907: [TASK] Auto-register set labels from labels.xlf | https://review.typo3.org/c/Packages/TYPO3.CMS/+/85907

**Review by linawolf — APPROVED** [review]

**Comment by garvinhicking** [comment]
Thank you for addressing it :)

**Pull request merged**
